packages/core/src/util.ts

```diff
@@ -7,7 +8,9 @@ export function loadImage(path: string) {
 }
 
 export function generateImageID(url: URL, config: ImageConfig) {
-  const baseURL = url.host ? new URL(url.origin + url.pathname) : new URL(url.protocol + url.pathname)
+  const baseURL = url.host
+    ? new URL(url.origin + url.pathname)
+    : new URL(url.protocol + path.relative(process.cwd(), url.pathname))
```

Comment:
i'm confused as to what this line is doing. I think `url.protocol` is something like `http:` and `path.relative` will return something like `static/image.png` and thus you'll end up with a URL like `https:static/image.png` which would be an invalid URL, right?

Reply:
It would be an invalid URL but I don't think it matters. It just needs to be a string which is unique for each image which is then hashed to generate an ID string. The change here is that instead of hashing a string containing an absolute file path to the image, we're hashing a string containing a relative path to the image.